sentinel-core/src/main/java/com/alibaba/csp/sentinel/slots/system/SystemRuleManager.java

```diff
@@ -89,7 +94,7 @@
     static {
         checkSystemStatus.set(false);
         statusListener = new SystemStatusListener();
-        scheduler.scheduleAtFixedRate(statusListener, 5, 1, TimeUnit.SECONDS);
+        scheduler.scheduleAtFixedRate(statusListener, 5, 100, TimeUnit.MILLISECONDS);
```

Comment:
100ms seems too frequent and might affect the performance. Maybe we can adjust it to 500ms or 1s?